fibonacci/script.js

```
@@ -0,0 +1,31 @@
+//Using iteration
+function fibs(num) {
+  const array = [];
+
+  let fib1 = 0;
+  let fib2 = 1;
+
+  for (let i = 0; i < num; i++) {
+    const newNum = fib1 + fib2;
+    fib1 = fib2;
+    fib2 = newNum;
+    array.push(newNum);
+  }
+
+  console.log(array);
+}
+//fibs(5); // [1, 2, 3, 5, 8]
+
+//Using Recursion
+
+function fibsRec(n) {
+  return n <= 0
+    ? "Please enter a positive number"
```

Throwing an error would be better for invalid cases instead of returning a completely different type